src/utils/postgres-client/index.ts

```diff
@@ -0,0 +1,379 @@
+import { v4 as uuidv4 } from 'uuid';
+import { User } from '@/types';
+import { SessionData } from 'express-session';
+import { Pool } from 'pg';
+export * from './types';
+
+import { ENV } from '../env';
+import { hashRefreshToken, newRefreshExpiry } from '../refresh-token';
+import { SqlUser, UserSecurityKey } from './types';
+import {
+  cameliseUser,
+  createUserQueryByColumn,
+  createUserQueryWhere,
+  getUserById,
+  insertUserRoles,
+  snakeiseUser,
+} from './utils';
+
+const pool = new Pool({
+  max: 20,
```

Comment:
I would suggest having this number configurable and default to 50 ([same as Hasura's](https://hasura.io/docs/latest/api-reference/syntax-defs/#pgpoolsettings)).

Reply:
I changed to 50, but I would rather not add a configuration until we need to use different values in our cloud, or someone asks for it (we already defined more than 100 environment variables, that's a crowd)

Reply:
@plmercereau sounds good :+1: